### PR TITLE
Russian translation

### PR DESCRIPTION
**Pull request by alexxwiz**
Hi!

I translated Cachet to russian. Can you merge it?

### CODE REVIEW

File: resources/lang/ru/forms.php [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+return [
+
+    // Setup form fields
+    'setup' => [
+        'email'            => 'Email',
```
Comment (ApfelUser) [inline resources/lang/ru/forms.php]: Missing translation

File: resources/lang/ru/forms.php [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+        'username'         => 'Имя пользователя',
+        'password'         => 'Пароль',
+        'site_name'        => 'Название сайта',
+        'site_domain'      => 'Домен',
+        'site_timezone'    => 'Выберите временную зону',
+        'site_locale'      => 'Выберите язык',
+        'enable_google2fa' => 'Включить двухфакторную авторизацию через Google',
+        'cache_driver'     => 'Cache Driver',
```
Comment (ApfelUser) [inline resources/lang/ru/forms.php]: Missing translation

File: resources/lang/ru/forms.php [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+        'session_driver'   => 'Session Driver',
```
Comment (ApfelUser) [inline resources/lang/ru/forms.php]: Missing translation

File: resources/lang/ru/forms.php [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+    ],
+
+    // Login form fields
+    'login' => [
+        'email'         => 'Email',
```
Comment (ApfelUser) [inline resources/lang/ru/forms.php]: Missing translation

File: resources/lang/ru/validation.php [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+    'regex'                => ':attribute формат неверен.',
+    'required'             => 'Поле :attribute является обязательным.',
+    'required_if'          => 'Поле :attribute является обязательным, если :other имеет значение :value.',
+    'required_with'        => 'Поле :attribute является обязательным, если заполнено :values.',
+    'required_with_all'    => 'Поле :attribute является обязательным, если :values заполнены.',
+    'required_without'     => 'Поле :attribute является обязательным, если :values не заполнены.',
+    'required_without_all' => 'Поле :attribute является обязательным, если ни одно из :values не заполнены.',
+    'same'                 => 'Поля :attribute и :other должны совпадать.',
+    'size'                 => [
+        'numeric' => 'The :attribute must be :size.',
```
Comment (ApfelUser) [inline resources/lang/ru/validation.php]: Missing translation

File: resources/lang/ru/validation.php [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+        'file'    => 'The :attribute must be :size kilobytes.',
```
Comment (ApfelUser) [inline resources/lang/ru/validation.php]: Missing translation

File: resources/lang/ru/validation.php [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+        'string'  => 'The :attribute must be :size characters.',
```
Comment (ApfelUser) [inline resources/lang/ru/validation.php]: Missing translation

File: resources/lang/ru/validation.php [hunk 8]
```diff
@@ -0,0 +1,116 @@
...
+        'array'   => 'The :attribute must contain :size items.',
```
Comment (ApfelUser) [inline resources/lang/ru/validation.php]: Missing translation

File: resources/lang/ru/validation.php [hunk 9]
```diff
@@ -0,0 +1,116 @@
...
+    ],
+    'string'               => 'The :attribute must be a string.',
```
Comment (ApfelUser) [inline resources/lang/ru/validation.php]: Missing translation

File: resources/lang/ru/validation.php [hunk 10]
```diff
@@ -0,0 +1,116 @@
...
+    'unique'               => 'The :attribute has already been taken.',
```
Comment (ApfelUser) [inline resources/lang/ru/validation.php]: Missing translation

**Comment by ApfelUser** [comment]
Hey, thanks for your pull request! I marked some missing translations, could you maybe add them?

**Comment by jbrooksuk** [comment]
Thanks for going through this @ApfelUser.

**Comment by alexxwiz** [comment]
Fixed strings. Also in CrowdIn.com

**Comment by ApfelUser** [comment]
:+1:

**Comment by jbrooksuk** [comment]
Thanks to you both @alexxwiz and @ApfelUser for this PR :)

**Comment by jbrooksuk** [comment]
@alexxwiz could you email me at james@cachethq.io :)

**Comment by ApfelUser** [comment]
@jbrooksuk I did.

**Comment by alexxwiz** [comment]
@jbrooksuk Done too.

**Comment by jbrooksuk** [comment]
Cheers guys :)

**Pull request merged**